frontend/src/components/containers/ContainersDetailContent.js

```diff
@@ -47,30 +71,39 @@ const ContainersDetailContent = ({containersByID, samplesByID, usersByID, get, l
           </Space>
       } />
-      <PageContent loading={!isLoaded && isFetching}>
-        <Title level={2}>Overview</Title>
-        <Descriptions bordered={true} size="small">
-          <Descriptions.Item label="Name" span={2}>{container.name}</Descriptions.Item>
-          <Descriptions.Item label="Barcode">{container.barcode}</Descriptions.Item>
-          <Descriptions.Item label="Location" span={2}>
-            {container.location ?
-              <Link to={`/containers/${container.location}`}>
-                {withContainer(containersByID, container.location, container => container.barcode, "Loading...")}
-              </Link>
-              : "—"}
-            {container.coordinates && ` at ${container.coordinates}`}
-          </Descriptions.Item>
-          <Descriptions.Item label="Kind">{container.kind}</Descriptions.Item>
-          <Descriptions.Item label="Comment" span={3}>{container.comment}</Descriptions.Item>
-        </Descriptions>
-
-        <TrackingFieldsContent entity={container}/>
-
-        <Descriptions bordered={true} size="small" title="Content Details" style={{marginTop: "24px"}}>
-          <Descriptions.Item span={3}>
-            <ContainerHierarchy key={id} container={isLoaded ? container : null} />
-          </Descriptions.Item>
-        </Descriptions>
+      <PageContent loading={!isLoaded && isFetching} style={pageStyle}>
+        <Tabs defaultActiveKey="1" size="large" type="card" style={tabsStyle}>
+          <TabPane tab="Overview" key="1" style={tabStyle}>
+              <Descriptions bordered={true} size="small">
+                <Descriptions.Item label="Name" span={2}>{container.name}</Descriptions.Item>
+                <Descriptions.Item label="Barcode">{container.barcode}</Descriptions.Item>
+                <Descriptions.Item label="Location" span={2}>
+                  {container.location ?
+                    <Link to={`/containers/${container.location}`}>
+                      {withContainer(containersByID, container.location, container => container.barcode, "Loading...")}
+                    </Link>
+                    : "—"}
+                  {container.coordinates && ` at ${container.coordinates}`}
+                </Descriptions.Item>
+                <Descriptions.Item label="Kind">{container.kind}</Descriptions.Item>
+                <Descriptions.Item label="Comment" span={3}>{container.comment}</Descriptions.Item>
+              </Descriptions>
+
+            <TrackingFieldsContent entity={container}/>
+
+            <Descriptions bordered={true} size="small" title="Content Details" style={{marginTop: "24px"}}>
+              <Descriptions.Item span={3}>
+                <ContainerHierarchy key={id} container={isLoaded ? container : null} />
+              </Descriptions.Item>
+            </Descriptions>
+          </TabPane>
+
+          <TabPane tab={`Experiments (${container.experiment_runs?.length})`} key="2" style={tabStyle}>
```

Comment:
Make tab optional using container_kind.is_run_container ?

Reply:
I am not a fan of having information displayed or hidden according to some business logic, without much explanations. I'd rather have a compromise between the two where it may be hidden but it is highlighted that it's because it's not a run container.. or have it displayed with a note that it's empty due to it not being a run container... will think about it